src/Form/Type/Filter/ChoiceType.php

```diff
@@ -28,10 +28,19 @@
  */
 class ChoiceType extends AbstractType
 {
+    /**
+     * @deprecated since 3.56, to be removed with 4.0: Use ContainsOperatorType const instead
```

Comment:
```suggestion
     * @deprecated since 3.x, to be removed with 4.0: Use ContainsOperatorType const instead
```
[The deprecation messages should be like that](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/CONTRIBUTING.md#deprecation) ([they will be changed when release](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/CONTRIBUTING.md#upgrading-code-comments-and-deprecation-messages)).

Reply:
Done